packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -26,12 +26,13 @@ import {
 const globalScope = (typeof window === 'undefined' ? globalThis : window) as any;
 const evalCode = globalScope[atob('ZXZhbA==')] as <T>(source: string) => T;
 
-let hasEval: boolean;
-try {
-  hasEval = evalCode<boolean>('true');
-} catch (_: unknown) {
-  hasEval = false;
-}
+const getHasEval = () => {
+  try {
+    return evalCode<boolean>('true');
+  } catch (_: unknown) {
+    return false;
+  }
+};
```

Comment:
Is it possible to avoid making the check everytime it's needed? Check once, store the value.

Reply:
Good idea it was indeed checked uselessly often. Thanks I changed it.